### DISCUSSION TITLE
How is this even possible?

### DISCUSSION THREAD

**Discussion by Akshansh16** [YimMenu/YimMenu#1518] [Research]
Players spawning vehicles like these. How is this possible with the menu?

![IMG_20230624_124504.jpg](https://github.com/YimMenu/YimMenu/assets/97584101/9e7dd698-ed83-4078-bb40-903a16661486)

**Comment by Tayipc38** [comment]
Those are just attachments to a certain vehicle. You can attach vehicles, peds and objects to make such creations. With a simple addition to the menu we could load xml files from menyoo in online.

**Reply by Akshansh16** [reply]
Menyooo is not accessible online. How can I access it?

**Reply by Yimura** [reply]
This is not YimMenu unless someone made this as a json vehicle.

**Reply by Tayipc38** [reply]
You can't access menyoo. What I mean was; Menyoo can create xml files that contain data relating to attachments as you see in those pictures. This data can be used to recreate those things but it would need to be implemented in YimMenu. Which I just might.
